src/Command/GenerateObjectAclCommand.php

```diff
@@ -50,14 +51,22 @@ class GenerateObjectAclCommand extends QuestionableCommand
     private $aclObjectManipulators = [];
 
     /**
-     * @var RegistryInterface
+     * @var RegistryInterface|ManagerRegistry|null
      */
     private $registry;
 
+    /**
+     * @param Pool $pool
+     * @param array $aclObjectManipulators
+     * @param RegistryInterface|ManagerRegistry|null $registry
+     */
     public function __construct(Pool $pool, array $aclObjectManipulators, $registry = null)
     {
         $this->pool = $pool;
         $this->aclObjectManipulators = $aclObjectManipulators;
+        if (!(method_exists($registry, 'getAliasNamespace') || method_exists($this->registry, 'getEntityNamespace'))) {
+            throw new \InvalidArgumentException('$registy need to be either Symfony\Bridge\Doctrine\RegistryInterface or Doctrine\Common\Persistence\ManagerRegistry');
```

Comment:
IMO, [`\TypeError`](https://www.php.net/manual/es/class.typeerror.php) is more suitable for this exception.
I'd say "argument 3" instead of referencing the argument name.
BTW, please use the `class` constant from these classes instead of hardcoding their QCNs.